app/views/jobs/index.html.erb

```diff
@@ -1,5 +1,21 @@
-All Jobs for <strong><%= @company.name %></strong>
+<%= link_to "Add a New Job", new_company_job_path(@company), class: "btn btn-success pull-right" %>
+<%= link_to "Regular Sort", company_jobs_path(@company), class: "btn btn-primary" %>
+<%= link_to "Sort by Interest", "#{company_jobs_path}/?sort=interest", class: "btn btn-primary" %>
```

Comment:
an alternative to this could be `company_jobs_path(sort: "interest)` - this will add a key value pair after the ? in our url 👍